Sources/Sentry/SentrySwizzle.m

```diff
@@ -102,11 +104,59 @@ @implementation SentrySwizzle
 
     pthread_mutex_lock(&gLock);
 
+#if TEST
+    IMP currImp = class_getMethodImplementation(classToSwizzle, selector);
+    [GULSwizzlingCache cacheCurrentIMP:currImp
+                             forNewIMP:newIMP
+                              forClass:classToSwizzle
+                          withSelector:selector];
+#endif
+
     originalIMP = class_replaceMethod(classToSwizzle, selector, newIMP, methodType);
 
     pthread_mutex_unlock(&gLock);
 }
 
+static void
+unswizzle(Class classToSwizzle, SEL selector)
+{
+#if TEST
+    static pthread_mutex_t gLock = PTHREAD_MUTEX_INITIALIZER;
```

Comment:
`m`: This code is copied from https://github.com/google/GoogleUtilities/blob/797005ad8a1f0614063933e2fa010a5d13cb09d0/GoogleUtilities/SwizzlerTestHelpers/GULSwizzler%2BUnswizzle.m. Why don't we also copy this class?

Reply:
It's not really a class, but an extensions of the `GULSwizzler`, which we don't use. Coping just a few lines sees easier. However I could include the reference.